File: CarsMatterApp/app/src/main/java/com/andrey/carsmatter/models/RefillNote.java
package com.andrey.carsmatter.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.Date;

public class RefillNote{
    @SerializedName("id")
    @Expose
    public int Id;

    @SerializedName("location")
    @Expose
    public String Location;

    @SerializedName("petrol")
    @Expose
    public float Petrol;

    @SerializedName("odo")
    @Expose
    public int Odo;

    @SerializedName("price")
    @Expose
    public float Price;

    @SerializedName("date")
    @Expose
    public Date Date;
}

File: CarsMatter.Infrastructure/Models/Journal/RefillNote.cs
using System;

namespace CarsMatter.Infrastructure.Models.Journal
{
    public class RefillNote
    {
        public int Id { get; set; }

        public string Location { get; set; }

        public float Petrol { get; set; }

        public int Odo { get; set; }

        public float Price { get; set; }

        public DateTime Date { get; set; }

    }
}
